static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -44,6 +44,11 @@ export type ReplayEventParameters = {
     seconds: number;
     user_email: string;
   };
+  'replay.erroroneous-table-navigate-to-details': {
+    platform: string | undefined;
+    project_id: string | undefined;
+    referrer: string;
+  };
```

Comment:
The params for `replay.erroroneous-table-*` and `replay.rage-dead-table-*` are the same as the original `replay.list-navigate-to-details`... hrm.

You can experiment with the existing events in amplitude here: https://app.amplitude.com/analytics/sentry/chart/new/x4oar5gu

The `referrer: string` field has a value like `/replay/` or `/issues/:groupId/replays/` which matches the url that we're looking at. What we want is, within that url, what specific table are we looking at...


So what if we added a new field to the original event, something like. `instance` or `table` or `referrer_table`. The values of the new field would be something like: `main | errors-table | dead-click-table`.

Then we can re-use the existing event, and support more tables in the future without having to copy+paste.

Reply:
good point! will implement